src/Umbraco.Core/Models/PublishedContent/IPublishedPropertyType.cs

```diff
@@ -65,6 +65,14 @@ public interface IPublishedPropertyType
     /// </remarks>
     Type ModelClrType { get; }
 
+    /// <summary>
+    ///     Gets the property model Delivery Api CLR type.
+    /// </summary>
+    /// <remarks>
+    ///     <para>The model CLR type may be a <see cref="ModelType" /> type, or may contain <see cref="ModelType" /> types.</para>
+    /// </remarks>
+    Type DeliveryApiModelClrType { get; }
```

Comment:
Adding a default implementation directly in the interface makes this a non-breaking change:

```suggestion
    Type DeliveryApiModelClrType => ModelClrType;
```